lbrynet/daemon/Daemon.py

```diff
@@ -988,6 +991,10 @@ def jsonrpc_status(self, session_status=False, dht_status=False):
         local_height = self.session.wallet.network.get_local_height() if has_wallet else 0
         remote_height = self.session.wallet.network.get_server_height() if has_wallet else 0
         best_hash = (yield self.session.wallet.get_best_blockhash()) if has_wallet else None
+        if has_wallet and self.session.wallet.wallet:
+            wallet_is_encrypted = self.session.wallet.wallet.use_encryption
+        else:
+            wallet_is_encrypted = False
```

Comment:
these 4 lines can just be 

    wallet_is_encrypted = has_wallet and self.session.wallet.wallet and 
        self.session.wallet.wallet.use_encryption